clinical/src/main/kotlin/com/hartwig/actin/clinical/curation/config/CurationConfigValidation.kt

```diff
@@ -137,6 +161,33 @@ inline fun <reified T : Enum<T>> validateEnum(
     }
 }
 
+fun validateIgnore(
```

Comment:
I think this function is specific to ComplicationConfigFactory so best to keep it there.  The other functions in this class are shared across multiple factories.